bin/cdnjs-index.js

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env node
+global._ = require('lodash'); // ¯\_(ツ)_/¯
+
+const path = require('path');
+const tar = require('tar-stream');
+const config = require('config');
+const zlib = require('zlib');
+const got = require('got');
+const micromatch = require('micromatch');
+const Bluebird = require('bluebird');
+
+const pipeline = require('util').promisify(require('stream').pipeline);
+
+const db = require('../src/lib/db/index.js');
+const CndJsPackage = require('../src/models/CdnJsPackage');
+
+const httpClient = got.extend({ headers: { 'user-agent': config.get('server.userAgent') }, json: true });
+
+const tarballUrl = 'https://github.com/cdnjs/packages/tarball/master';
+const versionedListUrl = 'https://api.cdnjs.com/libraries/?fields=version';
```

Comment:
I only now realized that even though we keep all name/version pairs in the DB, the script always fetches only the latest version. That means the results will depend on when exactly we run it, which is bad. E.g., if we run once a day and the project makes two releases on the same day, we miss one.

We should always query all versions (so yes, the initial fetch will be slow). Also, the "load all objects from DB at once" approach may need to change depending on how much data this creates (but that's a good thing cause the data would keep growing even with the current approach).